src/Interface/HasOrder.agda

```diff
@@ -2,31 +2,142 @@
 
 module Interface.HasOrder where
 
+open import Prelude hiding (_⊔_; suc; isPartialOrder; isEquivalence; trans)
 open import Level using (_⊔_; suc)
 open import Relation.Binary using (Rel)
-open import Relation.Binary.Definitions using (Decidable; Antisymmetric)
-open import Relation.Binary.Structures using (IsPreorder; IsPartialOrder)
+open import Relation.Binary.Definitions using (Decidable; Antisymmetric; Irreflexive; Asymmetric)
+open import Relation.Binary.Structures using (IsPreorder; IsPartialOrder; IsStrictPartialOrder; IsEquivalence)
```

Comment:
Personally, I think `Prelude`s should be minimal or non-existent.  I prefer explicit imports in the place where they're used.  I realize that's not the approach taken in the fls, but in cases like these, I'd would prefer to leave imports where they're used.  However, if you or Andre decide the list of imports is getting too "busy" and you want to move them, go for it. 😃

Reply:
We used to not have a `Prelude`, but the import lists just grow too long and become just noise entirely.

Also, note the top comment in `Relation.Binary.Structures`:
```-- The contents of this module should be accessed via `Relation.Binary`.```

Ignoring this can become an issue, because the interface provided by `Relation.Binary` is expected to be much more stable than `Relation.Binary.Definitions/Structures`, so there's a higher chance that upgrading our dependencies needs changes there.